sharkfuser/templates/external_tools.h.in

```diff
@@ -0,0 +1,2 @@
+// Auto-generated from external_tools.h.in - do not edit
```

Comment:
headers (I know it might seem frivolous here but this is the one thing that's free and makes life much easier not having to think about adding / not adding)